### PR TITLE
Add title and text to title slide

### PR DESCRIPTION
**Pull request by anseljh**

### CODE REVIEW

**Comment by anseljh** [comment]
I don't know why, but the deployment above doesn't show the image. Seems to not be there. See: https://trump-timeline-pr-15.herokuapp.com/images/l4gg-logo-500.png (gets an error) Any ideas, @jlegrone?

File: src/title-slide/slide-text.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<div style="display:block; float:right;"><img src="/images/l4gg-logo-500.png"></div>
```
Comment (jlegrone) [inline src/title-slide/slide-text.html]: This image isn't getting picked up by webpack; I suggest using a "media url".  Take a look at the SlideMedia argument here: https://github.com/L4GG/timeline/blob/b3c48db09589c009f6f8b6d9c9d57f85c2efefd9/src/utils/eventDirectoryToSlide.js#L52

I can commit a patch directly to this branch if you'd like.

File: src/title-slide/index.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+import SlideMedia from 'models/SlideMedia'
 
 import title from './slide-title.html'
 import text from './slide-text.html'
 
 const titleSlide = new Slide(
 	null,
 	new SlideContent(title, text),
-	null,
+	new SlideMedia('/images/l4gg-logo-500.png'),
```
Comment (jlegrone) [inline src/title-slide/index.js]: The full path isn't needed here; just `l4gg-logo-500.png`

Comment (jlegrone) [inline src/title-slide/index.js]: Wrong advice -- this should do it though:

```javascript
import image from 'images/l4gg-logo-500.png'

const titleSlide = new Slide(
	null,
	new SlideContent(title, text),
	new SlideMedia(image),
	null
)
```

**Pull request merged**